Checkout/Source/Model/SecurityCodeTokenDetails.swift

```diff
@@ -0,0 +1,14 @@
+// 
+//  File.swift
+//  
+//
+//  Created by Okhan Okbay on 10/10/2023.
+//
+
+import Foundation
+
+public struct SecurityCodeTokenDetails {
```

Comment:
This is called `CVVTokenDetails` in the case of Android. We should aim to keep things common whichever we can and if possible.